modules/apps/translation/translation-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -25,7 +25,9 @@ ViewDisplayContext viewDisplayContext = (ViewDisplayContext)request.getAttribute
 	propsTransformer="js/TranslationManagementToolbarPropsTransformer"
 />
 
-<clay:container-fluid>
+<clay:container-fluid
+	cssClass="container-view"
```

Comment:
Hi @boton  or @ambrinchaudhary 

I added this to make the same visual than the DM and the WC. But it affect not only the assetLibrary page, the Site too.

can You check if it's ok or there is another way.

Thanks!

Reply:
Hey @AliciaGarciaGarcia, do we havee any requirement for the Site pages to not look like this? Otherwise I would say this is better for consistency, and I guess than the mayority of the pages in portal should have that class to have same margins. 
So for me, the change is ok!